src/main/java/com/erii/ui/ControlPanel.java

```diff
@@ -0,0 +1,153 @@
+            return;
+        }
+        // Assuming the name and description of the task are the same for simplicity
+        taskManager.addTask(taskManager.new Todo("Todo", description, priority));
+
+    }
+    
+    private static void addDeadlineTask(String input, TaskManager taskManager) {
+        String[] parts = input.split(" ?/by |  ?/ ?");
+        if(parts.length < 3 || parts[1].isEmpty() || parts[2].isEmpty()){
+            System.out.println("Incorrect format. Please ensure the task description is followed by '/by' and a deadline date, and then a priority value (e.g., 'submit report /by 2021-09-30 /SS').");
+            return;
+        }
+        String description = parts[0];
+        String by = parts[1];
+        TaskManager.Priority priority;
+        try{
+            priority = TaskManager.Priority.valueOf(parts[2].trim().toUpperCase());
+        }catch(IllegalArgumentException e){
+            System.out.println("Invalid priority. Please enter a valid priority value (SS, S, A, B, C, D, E).");
+            return;
+        }
+        taskManager.addTask(taskManager.new Deadline("Deadline", description, by, priority));
+    }
+    
+    private static void addEventTask(String input, TaskManager taskManager) {
+        String[] parts = input.split(" /from | /to |  ?/ ?");
+        if (parts.length < 4 || parts[1].isEmpty() || parts[2].isEmpty() || parts[3].isEmpty()) {
+            System.out.println("Incorrect format. Please ensure the task description is followed by '/from', a start date, '/to', an end date, and then a priority value (e.g., 'project meeting /from 2021-09-30 /to 2021-10-01 /S').");
+            return;
+        }
+        String description = parts[0];
+        String start = parts[1];
+        String end = parts[2];
+        TaskManager.Priority priority;
+        try{
+            priority = TaskManager.Priority.valueOf(parts[3].trim().toUpperCase());
+        }catch(IllegalArgumentException e){
+            System.out.println("Invalid priority. Please enter a valid priority value (SS, S, A, B, C, D, E).");
+            return;
+        }
+        taskManager.addTask(taskManager.new Event("Event", description, start, end, priority));
+    }
+    
+    private static void markTaskAsDone(String input, TaskManager taskManager) {
+        try {
+            int taskNumber = Integer.parseInt(input.substring(5)) - 1; // Adjust for zero-based index
+            if (taskNumber < 0 || taskNumber >= taskManager.listSize()) {
+                System.out.println("Task number is out of range. Please enter a valid task number.");
+                System.out.println("Current number of tasks: " + taskManager.listSize());
+                return;
+            }
+            taskManager.markTaskAsDone(taskNumber);
+        } catch (NumberFormatException e) {
+            System.out.println("Please enter a valid task number.");
+        }
+    }
+
+    private static String getAddress() {
```

Comment:
the name of this function is a little misleading perhaps? I understand it probably refers to how to 'address' someone, but maybe use a word like `salutation` or `userTitle` instead?